### PR TITLE
Document changes to template functions in v0.128.0

### PR DESCRIPTION
**Pull request by jmooring**
- New: css.TailwindCSS
- Deprecate/rename: resources.Babel => js.Babel
- Deprecate/rename: resources.PostCSS => css.PostCSS
- Deprecate/rename: resources.ToCSS => css.Sass

### CODE REVIEW

**Comment by jmooring** [comment]
This is obviously just a stub at this point:
<https://deploy-preview-2637--gohugoio.netlify.app/functions/css/tailwindcss/>

The deprecation/renaming stuff is complete.

**Comment by bep** [comment]
>Deprecate/rename: resources.ToCSS => css.SASS

Hmm... Maybe `Sass` would be more correct?

**Comment by jmooring** [comment]
> Maybe Sass would be more correct?

Yes, it would be.

**Comment by bep** [comment]
OK, I have 

1. Renamed SASS => Sass
2. Moved the aliases.

**Comment by jmooring** [comment]
I'm assuming the options map will look something like this:
<https://deploy-preview-2637--gohugoio.netlify.app/functions/css/tailwindcss/#options>

**Comment by bep** [comment]
Tailwind options:

```go
type TailwindCSSOptions struct {
	Minify        bool // Optimize and minify the output
	Optimize      bool //  Optimize the output without minifying
	

// Service `mapstructure:",squash"`
	// Enable inlining of @import statements.
	// Does so recursively, but currently once only per file;
	// that is, it's not possible to import the same file in
	// different scopes (root, media query...)
	// Note that this import routine does not care about the CSS spec,
	// so you can have @import anywhere in the file.
	InlineImports bool

	// When InlineImports is enabled, we fail the build if an import cannot be resolved.
	// You can enable this to allow the build to continue and leave the import statement in place.
	// Note that the inline importer does not process url location or imports with media queries,
	// so those will be left as-is even without enabling this option.
	SkipInlineImportsNotFound bool
}
```

File: content/en/functions/css/TailwindCSS.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+targetPath
+: (`string`) If not set, the transformed resource's target path will be the original path of the asset file.
+
+## Example
+
+```go-html-template
+{{ with resources.Get "css/main.css" }}
+  {{ $opts := dict "targetPath" "css/style.css" "minify" true }}
+  {{ with . | css.TailwindCSS $opts }}
+    {{ if eq hugo.Environment "development" }}
```
Comment (bep) [inline content/en/functions/css/TailwindCSS.md:53]: ... hugo.IsDevelopment ...

File: content/en/functions/css/TailwindCSS.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+target = 'css'
+{{< /code-toggle >}}
+
+Process the resource:
+
+```go-html-template
+{{ with resources.Get "css/main.css" }}
+  {{ $opts := dict "minify" true }}
+  {{ with . | css.TailwindCSS $opts }}
+    {{ if eq hugo.IsDevelopment }}
```
Comment (bep) [inline content/en/functions/css/TailwindCSS.md:70]: remove eq

Comment (jmooring) [reply content/en/functions/css/TailwindCSS.md:70]: Thanks for the catch.

**Comment by bep** [comment]
Merged in 012162e0d2ede071ad2b9637db5fb1dbd5995d52

**Pull request closed without merging**